lib/factory_bot/strategy/build.rb

```diff
@@ -6,6 +6,8 @@ def association(runner)
       end
 
       def result(evaluation)
+        evaluation.notify(:before_build, nil)
```

Comment:
Added only to the strategies that also trigger `after_build`, skipping `build_stubbed` (debatable) and `attributes_for`